Sources_v3/StreamChatUI/Common Views/CreateNewChannelButton.swift

```diff
@@ -0,0 +1,32 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+open class CreateNewChannelButton: UIButton, AppearanceSetting {
+    // MARK: - Overrides
+    
+    override open var intrinsicContentSize: CGSize {
+        .init(width: 44, height: 44)
+    }
```

Comment:
Same here.... btw - why is this 44x44 and the avatar view is 40x40 ? :)

Reply:
1. The avatar view is `40x40` in the designs
2. The `44` is the navigation bar height. As the only place the `CreateNewChannelButton` is shown is the navigation bar I did it the same size.

Reply:
fixed